sunpy/net/base_client.py

```diff
@@ -75,6 +74,7 @@ def response_block_properties(self):
         s : `set`
             List of strings, containing attribute names in the response blocks.
         """
+        set()
```

Comment:
What's this line here for (and does it actually do anything?)

Reply:
```suggestion
        return set()
```